styles/modules/_userPage.scss

```diff
@@ -155,4 +155,8 @@
       }
     }
   }
+
+  &.connectedPeersPage {
```

Comment:
Since we're going to get rid of the connected peers page, would it be better to just switch the floR class on the button to floL?

Then it will be visible and we won't need to remember to remove this style rule later.